RocketChat/Rocket.Chat.ReactNative
# [FIX] Delete Subs

@RocketChat/ReactNative


Test plan:
- You'll need two users
- Logged as user2 on your phone
- Send a message from user1 to user2
- Close the app of user2
- Delete user1 account (Admin -> Users)
- Open app of user2 again
- User1 DM should be deleted

### Notes
I've tried to do something similar to subs but's not working properly.
E.g.: When you leave a channel with your app closed this channel will persist always on your phone, and this removed subscription isn't sent by server, needs investigation.
